### ISSUE TITLE
[BUG] [FEATURE-REQUEST] Add "region" field to ACCOUNT-V1 response

### ISSUE THREAD

**Issue by NFeruch** [RiotGames/developer-relations#889]
I think it would be a good idea to include a "Region" field in the response to  
ACCOUNT-V1's `/riot/account/v1/accounts/by-riot-id/{gameName}/{tagLine}`

That way, if a user accidentally chooses the wrong region in a website's UI, that can be validated right away instead of when we actually need to use the region as a prefix to other API calls.

For example
```JSON
{
    "puuid": "c60G40BQ960VmDubHu7vywjPJNaImANj5Z06CIbV2yqll4T9-HihI5Oq9voTIoesSHcB1KQnXMrF8g",
    "gameName": "faker",
    "tagLine": "00900"
    + "region": "na1"
}
```

This makes sense to me, because we are allowed to make this account call from any region. (`There are three routing values for account-v1; americas, asia, and europe. You can query for any account in any region.`)

It would be great if we could rely on users to select the correct region always, but this isn't the case, and it would be helpful to be able to validate that straight away. This would also potentially remove the need for the user to specify/input their region at all, as they would only need to input their riot name + tagline, and we will infer the region from the result.

When we sign a user up to our website, we ask them for their riot name, riot tagline, and region. If someone select the wrong region for their account, then it's a little more complicated to validate that, since the account-v1 endpoint can be accessed from any region. It would be helpful if there were a region field in account-v1's response initially, so that we wouldn't have to make another API call to see if the user selected the correct region

I could just make another API call with that region to see if the user selected the correct one, but it would just be a lot simpler if "region" could be a field in the response.

For API rate limits, that would double the amount of requests we have to make for each player that registers. This isn't a problem currently, but at scale it theoretically is.

**Comment by NFeruch** [comment]
I see that the endpoint `/riot/account/v1/active-shards/by-game/{game}/by-puuid/{puuid}` exists, but only supports LoR and Valorant. I'm not entirely what this returns but would adding LoL to this be a potential solution?

**Comment by tisbells** [comment]
Hey there - I don't have a timeline just yet, but we do have some planned updates that will support getting LoL/TFT regions by puuid. I'll post here when we have more info on what that looks like and an expected timeframe.

**Comment by NFeruch** [comment]
@tisbells Thank you for all the hard work you and the team are doing!

**Comment by Seiori** [comment]
Hello is there any update in regards to this?

**Comment by DarkIntaqt** [comment]
Hi, this feature would actually be really useful. I am not sure if the following is worth it's own issue, but it would be resolved when this feature gets added:

I received several reports that players on my site ended up in the wrong region, as regions someone transferred from are still returning data (outdated, but still there). If you don't know the currently active region on top of that, there is no way for me to tell whether they even transferred or not. This account region check would help resolving these problems.

**Comment by lucasMesquitaBorges** [comment]
I know it's a long time after this transition but we finally got unblocked on this.

The support for active shard for lol/tft was added, but it's a new endpoint!

https://developer.riotgames.com/apis#account-v1/GET_getActiveRegion

Feel free to re-open or create a new ticket if needed!
